adding a fix to parsing_user_query (#34816)
demisto/content: Packs/FeedMISP/ReleaseNotes/1_0_33.md

#### Integrations

##### MISP Feed

> - Fixed an issue where ***fetch-indicators*** where the query didn't pull updates.

Reviewer: ```suggestion
- Fixed an issue in the ***fetch-indicators*** command where indicator updates were not pulled.
```
